### DISCUSSION TITLE
Scan OK but results not showing in details

### DISCUSSION THREAD

**Discussion by zerr0s** [artifacthub/hub#3954] [Q&A] [answered]
Hello,

I've added a chart registry and all charts in it have been discovered successfully. The security scan is ok, but I can't view the report in the chart details. I don't know why ?

Any idea please ?

**Comment by tegioz** [comment, marked as answer]
Hi @zerr0s

Some ideas 🙂

- Check the repositories in the control panel to see when the repository was last scanned
- Take a look at the scanner logs to make sure everything when fine
- Make sure your package details page isn't cached locally (we ask for it to be cached for a few minutes)

**Comment by zerr0s** [comment]
Ok, all is working well in trivy logs and in the scanner job. The last security scan display a time that is correct.

I'll try to clear all cache ... Thanks
